File: Script/Fibonacci_ricorsivo/Fibonacci.js
function fibonacci(n) {
    if (n <= 1) return n;
    return fibonacci(n-1) + fibonacci(n-2);
}

const args = process.argv.slice(2);
if (args.length !== 1) {
    console.log(`Uso: node fibonacci_rec.js <numero>`);
    process.exit(1);
}

const n = parseInt(args[0], 10);
console.log(`Fibonacci(${n}) = ${fibonacci(n)}`);

File: Script/Fibonacci_ricorsivo/Fibonacci.java
public class Fibonacci {
    public static int fibonacci(int n) {
        if (n <= 1) return n;
        return fibonacci(n-1) + fibonacci(n-2);
    }

    public static void main(String[] args) {
        if (args.length != 1) {
            System.out.println("Uso: java FibonacciRec <numero>");
            return;
        }
        int n = Integer.parseInt(args[0]);
        System.out.println("Fibonacci(" + n + ") = " + fibonacci(n));
    }
}
